src/components/EmailValidationSection.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from "react";
+import { Store } from "redux";
+import { State } from "../reducers/index";
+import { LibraryData } from "../interfaces";
+import EmailValidationForm from "./EmailValidationForm";
+
+export interface EmailValidationSectionProps {
+  library: LibraryData;
+  store: Store<State>;
+}
+
+export default class EmailValidationSection extends React.Component<EmailValidationSectionProps, {}> {
+  render(): JSX.Element {
+    let emailForms = ["contact", "help", "copyright"].map((x) =>
+      <li className="well">
+        <EmailValidationForm
+          store={this.props.store}
+          uuid={this.props.library.uuid}
+          email={`${x}_email`}
+          libraryInfo={this.props.library.urls_and_contact}
```

Comment:
I'm not sure if rendering one form three times with the same "hiddenName" and "hiddenValue" is a problem. Probably not but something I can look into later.

Reply:
I'm not finding anything online that suggests it would be a problem.  I can see how there would be issues if all the forms were somehow being submitted together, but since they only get submitted one at a time...

Reply:
Yeah, since each is its own form, there shouldn't be a problem for individual submissions. But I'm not sure how screenreaders handle that (if they read the whole page and see the same value three times or read individual forms and see the value once). In any case, nothing to worry about at the moment.